e2e/vm/vm_test.go

```diff
@@ -27,7 +27,7 @@ func TestVM(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// The VM should be spined up in SynchronizedBeforeSuite of e2e/container.
+	// The VM should be spinned up in SynchronizedBeforeSuite of e2e/container.
```

Comment:
minor nit: but if we're fixing some spelling, I think the tense is wrong: "The VM should be **spun** up.."